examples/redux/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "redux-example",
+  "description": "Testing Redux interactions with RNTL",
+  "version": "0.0.1",
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "expo": "~37.0.3",
+    "expo-updates": "~0.1.0",
```

Comment:
Let's try to be expo-independent. We don't need the apps to be runnable, only tests. We don't have capacity to keep all of that updated and upgrade all examples

Reply:
Expo has been expelled